src/accelerate/commands/config/cluster.py

```diff
@@ -129,8 +129,16 @@ def get_cluster_input():
     else:
         dynamo_backend = DynamoBackend.NO
 
+    use_mps = False
+    if distributed_type == DistributedType.NO:
+        use_mps = _ask_field(
+            "Do you want to use Apple Silicon GPUs via `mps` device backend? [yes/NO]:",
+            _convert_yes_no_to_bool,
+            default=False,
+            error_message="Please enter yes or no.",
+        )
```

Comment:
I thought we were removing this entirely? Here `use_mps` should be `not use_cpu` when the MPS device is available, `False` otherwise.

Reply:
`not use_cpu` won't work. As I mentioned offline, it would ask for `gpu_ids` if it is `not use_cpu` which isn't applicable for `mps`.

Reply:
I have trouble understanding why, since if you set `use_mps` to `not use_cpu and is_mps_available()` here, at line 463 `use_mps=True` will prevent the question about GPUs.

Reply:
Yup, makes sense, addressed it, thanks for the pointer